tbapy/models.py

```diff
@@ -35,3 +38,12 @@ def _model_class(class_name):
 Rankings = _model_class('Rankings')
 DistrictRanking = _model_class('DistrictRanking')
 Status = _model_class('Status')
+
+
+class LastModifiedDate:
+    def __init__(self, date_string):
+        self.date_string = date_string
+        self.date = datetime.strptime(date_string, '%a, %d %b %Y %H:%M:%S GMT')
+
+    def __bool__(self):
```

Comment:
Why would the date always be falsy?

Reply:
I set the date to be always falsy for ease of use when writing code to fetch a new value or receive a "Not Modified" and return a cached one. Because the date is falsy, you can do:
`value = tba.get_endpoint(if_modified_since=date) or cached_value`
where cached_value represents the previously stored value in a cache and date is the date of last modification given by the API in a previous request.